src/components/InstallFeedback.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import {
+  Surface,
+  Button,
+  Icon,
+  CreateIssueButton,
+  useTessen,
+} from '@newrelic/gatsby-theme-newrelic';
+
+const InstallFeedback = () => {
+  const [clicked, setClicked] = useState(false);
+  const [feedback, setFeedback] = useState(null);
+  const tessen = useTessen();
+
+  const handleClick = (feedbackType) => {
+    setClicked(true);
+    setFeedback(feedbackType);
+    tessen.track({
+      eventName: 'installFeedbackThumbClick',
+      category: `${feedbackType}FeedbackClick`,
+      path: location.pathname,
+    });
+  };
+
+  return clicked ? (
+    <FeedbackResponse feedbackType={feedback} />
+  ) : (
+    <Surface
+      base={Surface.BASE.PRIMARY}
+      css={css`
+        padding: 2rem;
+        margin-bottom: 2rem;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        border: 1px solid #afe2e3;
+        background-color: rgba(175, 226, 227, 0.05);
+      `}
+    >
+      <div
+        css={css`
+          display: flex;
+          flex-direction: row;
+
+          @media screen and (max-width: 1200px) {
```

Comment:
```suggestion
          @media screen and (max-width: 1500px) {
```

will get those thumbs up/down looking right